prismic/fragments.py

```diff
@@ -5,9 +5,12 @@
 
 from collections import namedtuple, defaultdict, OrderedDict
 import logging
-import cgi
 import re
 import datetime
+try:
```

Comment:
Do you think that it would be better to check the running version with something like `supervisor.compat` or keep an error handling?

Reply:
To be honest I don't see much value in introducing a `compat` file here, this is the only place the `escape` function is called, but it's your call.

I think this needs to be fixed urgently, the `prismic` package is incompatible with the two latest Python releases, which is not great.

Reply:
html.escape is present in all currently supported Python versions (it dates back to 3.2), and cgi.escape has been gone for 2 stable releases. The simplest solution would be to forgo the try/except altogether and just do `from html import escape`